### PR TITLE
Add lang attribute to /r/<subreddit> API

### PR DESCRIPTION
**Pull request by justcool393**
This pull request adds the `lang` attribute to the /r/subreddit.json API, allowing clients to get the language the subreddit has been set to.

IFTA post: https://www.reddit.com/r/ideasfortheadmins/comments/38vxh8/
ModSupport post: https://www.reddit.com/r/ModSupport/comments/3cglol/what_are_some_small_problems_with_moderation_that/csvd87k

### CODE REVIEW

File: r2/r2/lib/jsontemplates.py [hunk 1]
```diff
@@ -610,6 +610,7 @@ class LinkJsonTemplate(ThingJsonTemplate):
         gilded="gildings",
         hidden="hidden",
         is_self="is_self",
+        lang="lang",
```
Comment (bsimpson63) [inline r2/r2/lib/jsontemplates.py]: this is the Link template, should you be modifying the Subreddit template?

Comment (justcool393) [inline r2/r2/lib/jsontemplates.py]: I can't believe I messed that up. Added to subreddit template and removed from other in 55adf2e.

**Comment by bsimpson63** [comment]
:fish: looks good, thanks

**Comment by bsimpson63** [comment]
Merged as https://github.com/reddit/reddit/commit/c76a7eb605dd716213060bee3d8632caee524b7a

What's your reddit username so I can give you a trophy?

**Comment by justcool393** [comment]
/u/justcool393 is my reddit username.

**Pull request closed without merging**